### DISCUSSION TITLE
Share your yaml files?

### DISCUSSION THREAD

**Discussion by azanting** [velzend/create_ikohs_fan#6] [General]
Hello velzend,

can you share the configurations that are necassary for implementing the fan? I'm specially interested in the custom card that you show on your repo.

I've got the following and it works. But your example looks better.

`localtuya:
  - host: x.x.x.x
    device_id: xxxxxxxxxxxxxxxxx
    local_key: xxxxxxxxxxxxxxxxxxxxx
    friendly_name: Ceilingfan
    protocol_version: "3.3"
    entities:
      - platform: switch
        friendly_name: Light
        id: 20

      - platform: switch
        friendly_name: Fan
        id: 60

      - platform: switch
        friendly_name: Direction
        id: 63`

**Comment by deese** [comment]
That would be super great. 
I have a basic configuration working. Light works and I can turn on and off the fan, but setting the fan speed doesn't work.

I also started the modification for the card but I'm not able to get the fan speed setting.

<img width="495" alt="image" src="https://user-images.githubusercontent.com/590758/163692074-533e8d79-7338-4e7b-8978-c938bd0e2deb.png">

In any case, thanks a lot for your work, cause this helped a lot already :)

KR,

-D
